### PR TITLE
auto-generated docs

### PR DESCRIPTION
**Pull request by mjudeikis**
```release-note
NONE
```

AutoGenerated test commands docs

### CODE REVIEW

File: test/commands_test.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	err = yaml.Unmarshal(b, &prow)
+	if err != nil {
+		t.Error(err)
+	}
+
+	for _, c := range prow.Presubmits.OpenshiftOpenshiftAzure {
+		commands = append(commands, c.RerunCommand)
+	}
+
+	f, err := os.OpenFile("../docs/commands.md",
```
Comment (jim-minter) [inline test/commands_test.go]: use `os.Create()`?

Comment (ehashman) [reply test/commands_test.go]: Oh, yeah, this is it... opened with APPEND mode below is the source of the bug.

**Comment by jim-minter** [comment]
fixed
/lgtm

**Pull request merged**